pages/main.js

```diff
@@ -95,27 +91,27 @@ function createTable (dats, send) {
             </div>
           </div>
         </td>
-        <td class="cell-right">${dat.stats.peers}</td>
-        <td class="cell-right">
+        <td class="tr">${dat.stats.peers}</td>
+        <td class="tr">
           ${bytes(dat.stats.bytesTotal)}
         </td>
         <td>
           ${button({
             icon: 'open-in-finder',
             text: '',
-            klass: 'row-action',
+            cls: 'row-action',
             click: () => send('app:open', dat)
           })}
           ${button({
             icon: 'link',
             text: '',
-            klass: 'row-action',
+            cls: 'row-action',
```

Comment:
doesn't that mess with anything that tries to parse the source? `class` is a reserved keyword